inspire/base/static/js/settings.js

```diff
@@ -172,7 +172,7 @@ require.config({
       deps: ["jquery"]
     },
     "feedback" : {
```

Comment:
I am wondering, do we need the shim configuration? Or if ``js/feedback/feedback.js`` is already requirejs friendly (as it seems to be) we can omit it. You can see a discussion on this [here](http://stackoverflow.com/questions/10918063/how-to-make-a-jquery-plugin-loadable-with-requirejs)